libs/ledger-live-common/src/apps/logic.ts

```diff
@@ -191,7 +191,12 @@ export const reducer = (state: State, action: Action): State => {
 
     case "recover":
       return { ...state, currentError: null };
-
+    case "wiped":
+      // Reconciliate an already achieved wipe via single uninstall apdu
+      return {
+        ...state,
+        installed: [],
+      }
```

Comment:
```suggestion
      };
```

It's missing a last `;` ;)